cpp/DataStorm/clock/CMakeLists.txt

```diff
@@ -6,18 +6,22 @@ include(../../cmake/common.cmake)
 
 add_executable(reader Reader.cpp)
 target_link_libraries(reader PRIVATE Ice::Ice Ice::DataStorm)
-add_custom_command(TARGET reader POST_BUILD
-  COMMAND ${CMAKE_COMMAND} -E copy -t $<TARGET_FILE_DIR:reader>
-    $<TARGET_RUNTIME_DLLS:reader>
-    $<GENEX_EVAL:$<TARGET_PROPERTY:Ice::Ice,ICE_RUNTIME_DLLS>>
-  COMMAND_EXPAND_LISTS
-)
+if(WIN32)
```

Comment:
Assuming you can add comments to cmake files (not obvious since there aren't any), it would be nice to add a comment here to explain this special Windows-only action.